content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+        update.path(section, saleId, id),
+        input,
+        update.stringify
+    );
+}
+export namespace update
+{
+    export type Input = Primitive<ISaleReview.IStore>;
+    export type Output = Primitive<ISaleInquiry<ISaleReview.IContent>>;
+
+    export const METHOD = "PUT" as const;
+    export const PATH: string = "/consumers/:section/sales/:saleId/reviews/:id";
+    export const ENCRYPTED: Fetcher.IEncrypted = {
+        request: false,
+        response: false,
+    };
+
+    export function path(section: string, saleId: string, id: number): string
+    {
+        return `/consumers/${section}/sales/${saleId}/reviews/${id}`;
+    }
+    export const stringify = createStringifier<Input>();
+}
+```
+
+
+
+
+#### Swagger
+Building `Swagger` is also possible and even much powerful.
+
+Although I think the `Swagger` is a typical tool that torturing the client developers, the `nestia` also can build the `swagger.json` file. Even `Swagger` generator of the `nestia` is much powerful and convenient than the `@nestjs/swagger`. It doesn't require any type of the swagger comment or DTO decorator function and just using the pure interface type as DTO is possible.
+
+Looking at the [simple/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fsimple%2Fswagger.json) file, generated by the `nestia`, it seems perfect. Exact route method, path and parameters are constructed and DTO structures are exactly same with the pure interace type `ISaleArticleComment`. Also, comments written on the controller class method, DTO interface `ISaleArticleComment` and its properties are exactly revied on the `description` fields in the `swagger.json`.
+
+Looking at the another file [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fgeneric%2Fswagger.json), you can find that there isn't any problem even when the generic typed DTO interfaces or controller classes come. Furthermore, traveling the `description` properties of the [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Funion%2Fswagger.json), comments written on the generic interfaces and controllers are exactly revived in each route functions and schema definitions.
+
+  - View in the `Swagger Editor`
+    - [simple/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fsimple%2Fswagger.json)
+    - [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fgeneric%2Fswagger.json)
+    - [union/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Funion%2Fswagger.json)
+
+<img src="https://github.com/samchon/nestia/wiki/images/swagger-editor-comment.png" style="max-width: 100%" />
```

Comment:
```suggestion
You can take a look at some of the generated swagger files [here](https://github.com/samchon/nestia/tree/master/demo/) and take them to an [online swagger editor](https://editor.swagger.io) to see what the generated swagger looks like.
```

Too many links in my opinion. If someone is interested, they'll be able to copy paste and look at the generated files